components/siddhi-tryit/org.wso2.carbon.siddhi.tryit.ui/src/main/resources/web/siddhitryit/siddhiProcessor_ajaxprocessor.jsp

```diff
@@ -0,0 +1,221 @@
+<%--
+~ Copyright (c) 2015, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
@prabathabey 
Thanks for pointing out. Fixed in PR: https://github.com/wso2/carbon-event-processing/pull/179